tests/unit/amo/sagas/test_collections.js

```diff
@@ -288,14 +288,71 @@ describe(__filename, () => {
 
       _addAddonToCollection(params);
 
-      const expectedLoadAction = addonAddedToCollection({
+      const expectedAddedAction = addonAddedToCollection({
         addonId: params.addonId,
         collectionId: params.collectionId,
         userId: params.userId,
       });
 
-      const loadAction = await sagaTester.waitFor(expectedLoadAction.type);
-      expect(loadAction).toEqual(expectedLoadAction);
+      const addedAction = await sagaTester.waitFor(expectedAddedAction.type);
+      expect(addedAction).toEqual(expectedAddedAction);
+
+      const unexpectedFetchAction = fetchCurrentCollectionPage({
+        page: 1,
+        errorHandlerId: errorHandler.id,
+        slug: collectionSlug,
+        user: params.userId,
+      });
+
+      expect(sagaTester.wasCalled(unexpectedFetchAction.type)).toEqual(false);
+      mockApi.verify();
+    });
+
+    it('posts an add-on to a collection in edit mode', async () => {
```

Comment:
It took me a while to understand what this "edit mode" was.

Reply:
I updated the title of the test to hopefully be a bit better.